src/index.html

```diff
@@ -5,8 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Ylitse MentorApp - SOS-Lapsikylä</title>
-    <link rel="stylesheet" type="text/css" href="/static/styles/fonts.css"/>
-    <link rel="stylesheet" type="text/css" href="/static/styles/styles.css"/>
+		<link rel="stylesheet" type="text/css" href="/static/styles/fonts.css"/>
```

Comment:
This indentation style is different to the one used in src/login/index.html. Which one should we decide to use?